packages/server/tests/acceptance/erc20.spec.ts

```diff
@@ -251,10 +251,15 @@ describe('@erc20 Acceptance Tests', async function () {
                                     });
 
                                     it('reverts', async function () {
+                                     try {
                                         await Assertions.expectRevert(
                                             contract.connect(spenderWallet).transferFrom(tokenOwner, to, amount),
                                             Constants.CALL_EXCEPTION
                                         );
+                                     }catch(e){
+                                        expect(e.code).to.be.equal('UNPREDICTABLE_GAS_LIMIT');
```

Comment:
Q: why are we catching `UNPREDICTABLE_GAS_LIMIT` in multiple places?
That doesn't seen to be the aim of the tests. We can always add another test cases for this specific case

Reply:
The UNPREDICTABLE_GAS_LIMIT is an error that ethers.js wraps around the error returned from the relayClient.  The JsonRpc error returned by the relay is handled properly in a standard hardhat project.  The test was updated to extract the JSON RPC Relay error and validate it.